config/packages/serializer/Centreon/Acknowledgement.Acknowledgement.yml

```diff
@@ -31,6 +31,12 @@ Centreon\Domain\Acknowledgement\Acknowledgement:
                 - 'Default'
                 - 'ack_main'
                 - 'ack_event_list'
+        authorName:
```

Comment:
didn't we decide to name fields using snake_case?

Reply:
don't worry in the response it will be present as `author_name`